src/interfaces/IPoolManager.sol

```diff
@@ -102,7 +107,8 @@ interface IPoolManager is IProtocolFees, IERC6909Claims, IExtsload, IExttload {
         external
         returns (int24 tick);
 
-    /// @notice All operations go through this function
+    /// @notice All interactions with this contract go through this function. A caller that calls `unlock` must implement
```

Comment:
This is actually not true - Initialize and updateDynamicFee can be called without the unlockCallback.